inspacy/pipe.py

```diff
@@ -1,6 +1,6 @@
 import spacy
 from spacy.tokens import Span
-
+import re
```

Comment:
`re` should be added to `requirements.txt` so that it gets installed with other dependencies

Reply:
ok. i added re module in requirements file in dispatch-1 branch